src/main.test.cpp

```diff
@@ -2,12 +2,21 @@
 #include <gtest/gtest.h>
 #include <spdlog/sinks/null_sink.h>
 #include <spdlog/spdlog.h>
+#include <filesystem>
 
 #include "silo/common/log.h"
 
 int main(int argc, char* argv[]) {
+   try {
+      if (!std::filesystem::exists("testBaseData/exampleDataset")) {
+         throw std::runtime_error("must be run in root of repository");
+      }
+   } catch (std::exception& e) {
+      std::cerr << e.what() << std::endl;
```

Comment:
Maybe turn this into a log message:
`SPDLOG_ERROR(...)`

Reply:
Why do we need this try-catch block anyway? What's wrong with simply letting the exception through?